src/services/youtubePlayer.js

```diff
@@ -53,7 +53,19 @@ function changeVolume(val) {
   return player.setVolume(val);
 }
 
-function triggerOnVideoChange(){
+async function increaseVolume(val) {
+  const oldVolume = await player.getVolume();
+  const newVolume = Math.min(oldVolume + val, 100);
+  return player.setVolume(newVolume);
+}
+
+async function decreaseVolume(val) {
```

Comment:
Prefer value over val.
```suggestion
async function decreaseVolume(value) {
```